examples/library/uwsgi_log_plugin.py

```diff
@@ -30,7 +30,7 @@
 
 
 def import_from_uwsgi_log(filename):
-    fields = UWSGI_FIELDS.keys()
+    fields = list(UWSGI_FIELDS.keys())
```

Comment:
I think tox is a greate tool.
https://github.com/henriquebastos/rows/pull/1/files